client/src/components/ui/top-navtab.tsx

```diff
@@ -7,14 +7,15 @@ interface Tab {
 
 interface TopNavtabProps {
   tabs: Tab[];
+  isFixed?: boolean;
 }
 
-export default function TopNavtab({ tabs }: TopNavtabProps) {
+export default function TopNavtab({ tabs, isFixed = false }: TopNavtabProps) {
   const [activeTab, setActiveTab] = useState(tabs[0].name);
 
   const handleTabClick = (tabName: string) => {
     setActiveTab(tabName);
-    console.log(`Active tab is now: ${tabName}`);
+    // console.log(`Active tab is now: ${tabName}`);
```

Comment:
Remove, don't need to comment out code - we can look through past commits if we ever need the code snippet in the future.